<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>842</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>842</width>
    <height>600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Планер</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 229, 199)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Заметки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;
border-radius: 10</string>
    </property>
    <property name="text">
     <string>Заметки</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
border-color: rgb(255, 85, 0);
font: 12pt &quot;Fixedsys&quot;;
border-radius: 10</string>
    </property>
    <property name="text">
     <string>Календарь</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Fixedsys&quot;;</string>
    </property>
    <property name="text">
     <string>НЕДЕЛЯ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
    </property>
    <property name="text">
     <string>          Уведомления</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>207</y>
      <width>741</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 228);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 228);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_5">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 228);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_7">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 228);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_6">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 228);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 228);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 228);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 246, 228);
border-radius: 10;
font: 12pt &quot;Fixedsys&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Заметки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
font: 12pt &quot;Fixedsys&quot;;
border-radius: 10</string>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 246, 228);
border-radius: 10</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
